readthedocs/restapi/serializers.py

```diff
@@ -12,6 +12,15 @@
 
 class ProjectSerializer(serializers.ModelSerializer):
     canonical_url = serializers.ReadOnlyField(source='get_docs_url')
+    description = serializers.SerializerMethodField()
+
+    def get_description(self, obj):
+        """
+        Forcing the description field to be an empty string.
+
+        See https://github.com/rtfd/readthedocs.org/issues/3689.
```

Comment:
Forcing the description to be an empty string in the API response. This was done as per [this comment](https://github.com/rtfd/readthedocs.org/issues/3689#issuecomment-371686362).